source/INI/PeanutButter.INI/INIFile.cs

```diff
@@ -540,6 +540,7 @@ private List<string> GetLinesForCurrentData(PersistStrategies persistStrategy)
                     dictionary
                         .Keys
                         .Select(key => LineFor(section, key)));
+                lines.Add("-----");

```

Comment:
this would be incorrect: adding dashes on a line to an INI file is invalid syntax. The line would have to be commented with `;`, eg: add `; -----`

the spirit of the open issue is that this should be configurable -- and I'd tend towards the default being a blank line, not a line with data in it. I think that most people would expect to see something like this in a nicely-formatted ini file:

```csharp
[section1]
setting1=value1

[section2]
setting2=value2
```

but I can understand if someone specifically wanted to put something in that gap, hence the requirement for configurability -- perhaps as a string property on `IINIFile`, and if it's null or whitespace, simply add a new line.

Reply:
I'm the original requestor for this feature.
A single blank line would be perfect.  No need for configurability.